ember-cli-build.js

```diff
@@ -13,7 +13,12 @@ module.exports = function (defaults) {
         DEPRECATE_STORE_EXTENDS_EMBER_OBJECT: false,
       },
     },
-    // Add options here
+    autoImport: {
+      watchDependencies: ['mirage'],
```

Comment:
why do we need to do this? 🤔 this would suggest that we're editing miragejs itself 🤔